cinema/tests/test_movie_api.py

```diff
@@ -63,6 +63,7 @@ def test_get_movies_with_actors_filtering(self):
 
     def test_get_movies_with_title_filtering(self):
         movies = self.client.get(f"/api/cinema/movies/?title=ita")
+        print("Response data:", movies.data)
```

Comment:
The `print` statement is generally not recommended in test cases as it can clutter the test output. Consider removing it unless it's specifically needed for debugging purposes.